python/hsfs/feature_view.py

```diff
@@ -257,6 +257,8 @@ def init_serving(
         training_dataset_version: Optional[int] = None,
         external: Optional[bool] = None,
         options: Optional[dict] = None,
+        init_online_store_sql_client: Optional[bool] = None,
```

Comment:
Shouldn't we ideally set one of init_online_store_sql_client or init_online_store_rest_client as True by default so as to make it more easier for users.